interfaces/builtin/unity7.go

```diff
@@ -482,6 +482,8 @@ dbus (receive)
 # this leaks the names of snaps with desktop files
 /var/lib/snapd/desktop/applications/ r,
 /var/lib/snapd/desktop/applications/mimeinfo.cache r,
+# parallel-installs: this leaks read access to desktop files owned by keyed
+# instances of @{SNAP_NAME} to @{SNAP_NAME} snap
 /var/lib/snapd/desktop/applications/@{SNAP_NAME}_*.desktop r,
```

Comment:
This should be adjusted to be:
```
/var/lib/snapd/desktop/applications/@{SNAP_INSTANCE_NAME}_*.desktop r,
```

Let's make the commend slightly more clear with:
```
# parallel-installs: when @{SNAP_INSTANCE_NAME} == @{SNAP_NAME},
# this leaks read access to desktop files of parallel installs of the snap
```